app/assets/javascripts/users.js

```diff
@@ -1,7 +1,7 @@
-document.addEventListener("DOMContentLoaded", function(){
-  const button = document.getElementById('edit-button');
-	var display = button.dataset.display;
-	if (display === 'true'){
-		button.style.display = 'block';
-	}
-});
\ No newline at end of file
+// document.addEventListener("DOMContentLoaded", function(){

```

Comment:
Did you mean to include this as part of the commit?

Reply:
I didn't. I should address those event handlers separately.

Reply:
This has been restored